keyboards/rgbkb/sol3/rules.mk

```
@@ -0,0 +1,46 @@
+# MCU name
+MCU = STM32F303
+
+# Touch encoder needs
+SRC += ../common/touch_encoder.c
+SRC += ../common/common_oled.c
+QUANTUM_LIB_SRC += i2c_master.c
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = no       # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = no        # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = yes        # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+NKRO_ENABLE = yes           # Enable N-Key Rollover
+AUDIO_ENABLE = yes          # Audio output
+AUDIO_DRIVER = dac_basic
```

I'd recommend dac_additive instead.  dac basic seems to have some issues, especially for stuff like music mode and audio clicky. 
```suggestion
AUDIO_DRIVER = dac_additive
```